### PR TITLE
Microsoft.Compute: Add CloudServices in stable 2021-03-01 Version

### PR DESCRIPTION
**Pull request by stephenhou**
REFER HERE: https://github.com/Azure/azure-rest-api-specs/pull/13155

### CODE REVIEW

File: specification/compute/resource-manager/readme.md [hunk 1]
```diff
@@ -207,6 +207,11 @@ directive:
       - TrackedResourcePatchOperation
     reason:
       - ACS service is deprecated so a PATCH endpoint won't be implemented
+  - where:
+      - $.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/cloudServices/{cloudServiceName}/roleInstances/{roleInstanceName}/remoteDesktopFile"].get
+    suppress:
```
Comment (Sandido) [inline specification/compute/resource-manager/readme.md:212]: Why is this needed?

File: specification/compute/resource-manager/Microsoft.Compute/stable/2021-03-01/cloudService.json [hunk 2]
```diff
@@ -0,0 +1,2680 @@
...
+        "frontendIPConfigurations": {
+          "description": "Specifies the frontend IP to be used for the load balancer. Only IPv4 frontend IP address is supported. Each load balancer configuration must have exactly one frontend IP configuration.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/LoadBalancerFrontendIPConfiguration"
+          }
+        }
+      },
+      "required": [
+        "frontendIPConfigurations"
```
Comment (Sandido) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2021-03-01/cloudService.json:2111]: Making a previously optional property mandatory is a breaking change. 
Please remove the Required portion and make edits in the backend as necessary.

File: specification/compute/resource-manager/Microsoft.Compute/stable/2021-03-01/cloudService.json [hunk 3]
```diff
@@ -0,0 +1,2680 @@
...
+          "description": "The name of the Load balancer",
+          "type": "string"
+        },
+        "properties": {
+          "$ref": "#/definitions/LoadBalancerConfigurationProperties",
+          "description": "Properties of the load balancer configuration."
+        }
+      },
+      "required": [
+        "name",
```
Comment (Sandido) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2021-03-01/cloudService.json:2132]: Making a previously optional property mandatory is a breaking change.
Please remove the Required portion and make edits in the backend as necessary.

**Review by Sandido — CHANGES_REQUESTED** [review]

**Comment by Sandido** [comment]
If you edit the existing 2020-10-01-preview cloudservice.json file with the breaking changes and have another SDK release, you can then bring the breaking changes over to this stable release without issue. 
We do not want breaking changes comparing the Stable release to the latest Preview release. If the Preview release already has them, then that is not a breaking change.

File: specification/compute/resource-manager/readme.md (hunk 1 above)
Comment (stephenhou) [reply specification/compute/resource-manager/readme.md:212]: the reason is listed below: reason: The API response has binary format and file type which is valid Swagger format. However, the example must be a JSON file which does not support specifying this response format.

The team owning the tests told me to supress it as they don't have a solution

**Comment by stephenhou** [comment]
> If you edit the existing 2020-10-01-preview cloudservice.json file with the breaking changes and have another SDK release, you can then bring the breaking changes over to this stable release without issue.
> We do not want breaking changes comparing the Stable release to the latest Preview release. If the Preview release already has them, then that is not a breaking change.

Sounds good, that is being addressed by making changes to the preview version here: https://github.com/Azure/azure-rest-api-specs/pull/12666

File: specification/compute/resource-manager/Microsoft.Compute/stable/2021-03-01/cloudService.json (hunk 2 above)
Comment (stephenhou) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2021-03-01/cloudService.json:2111]: taking below suggestion

File: specification/compute/resource-manager/Microsoft.Compute/stable/2021-03-01/cloudService.json (hunk 3 above)
Comment (stephenhou) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2021-03-01/cloudService.json:2132]: taking below suggestion
